Toggle k-th bit of a number from Right in C++

mansi-9 opened the pull request:
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
 
Yes

### Description
Toggle k-th bit of a number from Right in C++

### Checklist

- [yes ] I've read the contribution guidelines.
- [ yes] I've checked the issue list before deciding what to submit.
- [ yes] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#2890

mansi-9 committed:
resolved issue #2890 i.e. toggle kth bit from right

mansi-9 committed:
#2890 toggle kth bit

#2890 toggle kth bit

mansi-9 commented:
@bhav09 kindly review

iamrajiv commented on C-Plus-Plus/README.md:
```diff
@@ -50,6 +50,7 @@
 - [Sub-array with given sum](cp/SubArrayWithGivenSum.cpp)
 - [Suduko Solver](cp/SudukoSolver.cpp)
 - [Target Sum triplets](cp/target_sum_triplets.cpp)
+- [Toggle Kth Bit From Right](ToggleKthBitFromRight.cpp)
```
@mansi-9 The path should be `- [Toggle Kth Bit From Right](cp/ToggleKthBitFromRight.cpp)`.

mansi-9 replied:
Sure. Thanx!!!

iamrajiv reviewed (changes requested):
@mansi-9 Add a description of the problem and link to the problem.

mansi-9 committed:
#2890 toggle kth bit from right

#2890 toggle kth bit from right

mansi-9 committed:
#2890 toggle kth bit of a number

#2890 Toggle kth bit from the right i.e.For a given number n, if k-th bit from right is 0, then toggle it to 1and if it is 1 then, toggle it to 0.

mansi-9 commented:
@iamrajiv did the changes sir.

iamrajiv committed:
formatted

iamrajiv commented:
@all-contributors please add @mansi-9 for her code
